engine/lib/tags.php

```diff
@@ -89,7 +87,6 @@ function elgg_get_tags(array $options = array()) {
 
 		'owner_guids' => ELGG_ENTITIES_ANY_VALUE,
 		'container_guids' => ELGG_ENTITIES_ANY_VALUE,
-		'site_guids' => $CONFIG->site_guid,
```

Comment:
May add `_elgg_validate_site_guid()` that will throw an exception if site guid is not current site guid, and use it in ege* functions?